docs/src/tutorial/genomic-surveillance.rst

```diff
@@ -0,0 +1,177 @@
+Run the workflow using a genomic surveillance configuration
+===========================================================
+
+In this tutorial, you will replace the custom data with meaningful focal data and use a configuration commonly found in genomic surveillance applications.
```

Comment:
Again (i'm a broken record!) I suggest we frame this in terms of aims:

> In the previous tutorial we showed how to analyse a small set of GISAID ("custom") data in the context of a small set of reference data. For genomic surveillance applications, we often have a set of data specific to our question of interest, for instance a set of sequences from a particular geographic area, which is referred to as the focal set. We want to analyse this focal set in a global context, and since there are millions of global sequences we want to subset these based on genomic proximity to our focal set.

Reply:
Thanks! I wouldn't have been able to write it that nicely so I appreciate you being a broken record here.

Done: a008ed21